src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -16,7 +16,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_AMOUNT = new Prefix("amt/");
     public static final Prefix PREFIX_OTHER_PARTY = new Prefix("o/");
     public static final Prefix PREFIX_DATE = new Prefix("dt/");
-
-
-
+    public static final Prefix PREFIX_START_MONTH = new Prefix("s/");
+    public static final Prefix PREFIX_END_MONTH = new Prefix("e/");
```

Comment:
Just a minor thing but would it be better if the prefix for end month is `end/` instead of `e/`? Since `e/` is already used as a prefix for email

Reply:
Implementation wise it doesn't matter whether the prefixes are repeated. Do you think it might confuse the user?

Reply:
Ok then I think I will leave it as it is 👍